apps/workflow-helper.sh

```diff
@@ -74,6 +74,11 @@ rm -f /sbin/mdev
 
 mkdir /worker
 
+# Certain workflow actions require access to the 'sg' device
+# We load the 'sg' module here to enable that, but should probably
+# load the module within the action(s) requring it
+modprobe sg
```

Comment:
I can see this being useful but this can also be a pattern of modprobing drivers that aren't actually necessary, and what happens if we have conflicting needs (not sure if realistic)? I think it might be better to use `cexec` + mounting host dir and then modprobing in there via the workflows